sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -65,6 +68,9 @@
     private static final Duration CONNECTION_ACQUIRE_TIMEOUT = Duration.ofSeconds(45);
     private static final int REACTOR_NETTY_MAX_CONNECTION_POOL_SIZE = 1000;
     private static final String REACTOR_NETTY_CONNECTION_POOL_NAME = "reactor-netty-connection-pool";
+    private static final int DEFAULT_DIRECT_HTTPS_RESPONSE_TIMEOUT_IN_SECONDS = 60;
+    private static final int DEFAULT_QUERY_PLAN_RESPONSE_TIMEOUT_IN_SECONDS = 3;
+    private static final int DEFAULT_ADDRESS_REFRESH_RESPONSE_TIMEOUT_IN_SECONDS = 3;
```

Comment:
NIT - do we also want to expose maxRetryCount in VMOptions?

Reply:
We decided to not expose it, as we have other properties in that file too which we are not exposing. So let's keep it as it is for now, if customers want it, we can expose it. 
@FabianMeiswinkel @simplynaveen20 @moderakh  thoughts?

Reply:
Yes lets not expose it  , retry is internal logic ,and we have not expose it for others either